chess_engine.py

```diff
@@ -118,6 +118,16 @@ def is_valid_piece(self, row, col):
         evaluated_piece = self.get_piece(row, col)
         return (evaluated_piece is not None) and (evaluated_piece != Player.EMPTY)
 
+    def get_white_king_location(self):
+        return self._white_king_location
+
+    def get_black_king_location(self):
+        return self._black_king_location
+
+    def is_valid_torn(self, row, col):
```

Comment:
Spelling mistake, 'torn' should be 'turn'

Reply:
I fixed it

Reply:
I fixed it